packages/react/src/Spinner/Spinner.dev.stories.tsx

```diff
@@ -7,4 +7,4 @@ export default {
   component: Spinner,
 } as Meta<typeof Spinner>
 
-export const Default = () => <Spinner sx={{border: '1px solid red'}} />
+export const Default = () => <Spinner sx={{border: '1px solid red'}} size={'small'} />
```

Comment:
For text-only values we can use this syntax instead:

```suggestion
export const Default = () => <Spinner sx={{border: '1px solid red'}} size="small" />
```